app/features/build_runner/build_runner.rb

```diff
@@ -460,7 +460,7 @@ def save_build_status_locally!
     def save_build_status_source!
       status_context = project.project_name
 
-      build_path = FastlaneCI::BuildController.build_url(project_id: project.id, build_number: current_build.number)
+      build_path = FastlaneCI::BuildJSONController.build_url(project_id: project.id, build_number: current_build.number)
       build_url = FastlaneCI.dot_keys.ci_base_url + build_path
```

Comment:
Side question, how does a User set up the `ci_base_url`?

Reply:
`FASTLANE_CI_BASE_URL` - check out https://github.com/fastlane/ci/blob/master/docs/EnvironmentVariables.md for more info 😄

Reply:
Seems pretty important, should this be a separate global setting?